manuscript.Rmd

```diff
@@ -240,7 +248,8 @@ $P(E|H)$ is the probability of obtaining evidence for a hypothesis when it holds
 We call this probability of detecting evidence, given that the hypothesis holds "detectability."
 Consequently, researchers should ensure that their study design allows them to find evidence for their hypothesis, in case it is true.
 When applied strictly within the bounds of null hypothesis testing, detectability is equivalent to power (or the complement of type-II error rate).
-However, while detectability is of great importance for study design, it is not directly relevant to the objective of preregistration.
+However, while detectability is of great importance for study design, it is not directly relevant to what a preregistration is comunicating to other researchers.
+However, we latter discuss how issues detectability must be considered for the issue of preregistration.
```

Comment:
There is two "howevers" in the two sentences. Also, I don't understand the second sentence.
Do you mean:
```suggestion
We later discuss how issues of detectability must be considered in a preregistration.
```